integrations/vault/sys/sys/src/main/java/module-info.java

```diff
@@ -23,6 +23,7 @@
 module io.helidon.integrations.vault.sys {
     requires java.logging;
 
+    requires io.helidon.common.reactive;
```

Comment:
This module exposes a public API `SysRx` that uses `Single`. We should use requires transitive. CC @tomas-langer

Reply:
Fixed